### DISCUSSION TITLE
Create Money Field

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#9807] [Ideas]
Hi everyone

One suggestion would be to create a Field Type Money. The Number field does not allow you to place mask with myo separator.

tks.

**Comment by melohagan** [comment]
Hey @Kakovisk 

It would be nice to have a custom money field plugin, but there is a input mask plugin at the moment which may do the job? https://github.com/melohagan/budibase-component-nikita-input-mask
